clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -359,8 +359,8 @@ private static int parseAcks(String acksString) {
      * records waiting to be sent. This allows sending many records in parallel without blocking to wait for the
      * response after each one.
      * <p>
-     * The result of the send is a {@link RecordMetadata} specifying the partition the record was sent to and the offset
-     * it was assigned.
+     * The result of the send is a {@link RecordMetadata} specifying the partition the record was sent to, the offset
+     * it was assigned and the timestamp of the record.
```

Comment:
Since this will be part of the java doc, could we explain what the timestamp will be?